assignments/models.py

```diff
@@ -121,3 +121,30 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class Student(models.Model):
+    assignment = models.ForeignKey(Assignment, related_name='students')
+    school = models.CharField(max_length=256)
```

Comment:
@frwickst  This is postgresql DB. Do you have any recommendations for CharField?